packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/rest-length/expected.js

```diff
@@ -0,0 +1,14 @@
+var t = function (f) {
+  arguments.length <= 1 ? undefined : arguments[1];
+  arguments.length <= arguments.length - 1 - 1 + 1 ? undefined : arguments[arguments.length - 1 - 1 + 1];
```

Comment:
Yeah the minifier should help, but we can also do some evaluation here. If
you want to try it, check out path.evaluate

On Wednesday, January 6, 2016, Kevin Decker <notifications@github.com>
wrote:

> In
> packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/rest-length/expected.js
> <https://github.com/babel/babel/pull/3246#discussion_r49040640>:
>
> > @@ -0,0 +1,14 @@
> > +var t = function (f) {
> > +  arguments.length <= 1 ? undefined : arguments[1];
> > +  arguments.length <= arguments.length - 1 - 1 + 1 ? undefined : arguments[arguments.length - 1 - 1 + 1];
>
> Could his be optimized out or is the assumption that uglify will reduce it
> back to the simplified args[args.length - 1] form?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/babel/babel/pull/3246/files#r49040640>.
>